2021-04-06 17:36:01,425 - [engine.py:774-_internal_run()] - ERROR - Engine run is terminating due to exception: name 'logger' is not defined
2021-04-06 17:36:44,962 - [test.py:41-log_metrics()] - INFO - Loss: 0.61635458984375 Accuracy: 0.8761 ConfusionMatrix: tensor([[ 915,    0,    2,    2,    0,    1,   57,    1,    2,    0],
        [   1, 1079,   13,    3,    2,    0,   24,    3,   10,    0],
        [   5,    5,  963,   16,    5,    3,   16,    9,   10,    0],
        [   2,    0,   29,  906,    3,   22,    4,   16,   27,    1],
        [   2,    6,    1,    5,  891,    3,   42,    4,    4,   24],
        [   8,    4,    0,   58,    6,  772,   33,    1,    8,    2],
        [   4,    5,    2,    0,    2,    8,  937,    0,    0,    0],
        [  19,    7,  120,   18,    5,    2,    0,  821,   13,   23],
        [   9,    6,   61,   21,   12,   22,   36,    2,  788,   17],
        [  21,    4,    5,   10,  192,   31,   11,   18,   28,  689]]) acc_0: 0.902 acc_1: 0.8878 acc_2: 0.8968 acc_3: 0.899 acc_4: 0.9018 acc_5: 0.9108 acc_6: 0.9042 acc_7: 0.8972 acc_8: 0.9026 acc_9: 0.8991 
2021-04-06 17:37:54,086 - [engine.py:694-run()] - INFO - Engine run starting with max_epochs=1.
2021-04-06 17:37:57,844 - [test.py:41-log_metrics()] - INFO - Loss: 0.61635458984375 Accuracy: 0.8761 ConfusionMatrix: tensor([[ 915,    0,    2,    2,    0,    1,   57,    1,    2,    0],
        [   1, 1079,   13,    3,    2,    0,   24,    3,   10,    0],
        [   5,    5,  963,   16,    5,    3,   16,    9,   10,    0],
        [   2,    0,   29,  906,    3,   22,    4,   16,   27,    1],
        [   2,    6,    1,    5,  891,    3,   42,    4,    4,   24],
        [   8,    4,    0,   58,    6,  772,   33,    1,    8,    2],
        [   4,    5,    2,    0,    2,    8,  937,    0,    0,    0],
        [  19,    7,  120,   18,    5,    2,    0,  821,   13,   23],
        [   9,    6,   61,   21,   12,   22,   36,    2,  788,   17],
        [  21,    4,    5,   10,  192,   31,   11,   18,   28,  689]]) acc_0: 0.902 acc_1: 0.8878 acc_2: 0.8968 acc_3: 0.899 acc_4: 0.9018 acc_5: 0.9108 acc_6: 0.9042 acc_7: 0.8972 acc_8: 0.9026 acc_9: 0.8991 
2021-04-06 17:37:57,844 - [engine.py:757-_internal_run()] - INFO - Epoch[1] Complete. Time taken: 00:00:04
2021-04-06 17:37:57,844 - [engine.py:770-_internal_run()] - INFO - Engine run complete. Time taken: 00:00:04
2021-04-06 20:40:42,715 - [engine.py:694-run()] - INFO - Engine run starting with max_epochs=1.
2021-04-06 20:40:44,566 - [test.py:42-log_metrics()] - INFO - Loss: 0.61635458984375 Accuracy: 0.8761 ConfusionMatrix: tensor([[ 915,    0,    2,    2,    0,    1,   57,    1,    2,    0],
        [   1, 1079,   13,    3,    2,    0,   24,    3,   10,    0],
        [   5,    5,  963,   16,    5,    3,   16,    9,   10,    0],
        [   2,    0,   29,  906,    3,   22,    4,   16,   27,    1],
        [   2,    6,    1,    5,  891,    3,   42,    4,    4,   24],
        [   8,    4,    0,   58,    6,  772,   33,    1,    8,    2],
        [   4,    5,    2,    0,    2,    8,  937,    0,    0,    0],
        [  19,    7,  120,   18,    5,    2,    0,  821,   13,   23],
        [   9,    6,   61,   21,   12,   22,   36,    2,  788,   17],
        [  21,    4,    5,   10,  192,   31,   11,   18,   28,  689]]) acc_0: 0.9279918864097363 acc_1: 0.9668458781362007 acc_2: 0.8051839464882943 acc_3: 0.8719923002887392 acc_4: 0.7969588550983899 acc_5: 0.8935185185185185 acc_6: 0.8077586206896552 acc_7: 0.9382857142857143 acc_8: 0.8853932584269663 acc_9: 0.9113756613756614 
2021-04-06 20:40:44,566 - [engine.py:757-_internal_run()] - INFO - Epoch[1] Complete. Time taken: 00:00:02
2021-04-06 20:40:44,567 - [engine.py:770-_internal_run()] - INFO - Engine run complete. Time taken: 00:00:02
